packages/blockchain-common/test/registries/FabricEnvironmentRegistry.test.ts

```diff
@@ -12,22 +12,24 @@
  * limitations under the License.
 */
 
-import { FabricEnvironmentRegistry } from '../../extension/registries/FabricEnvironmentRegistry';
+import { FabricEnvironmentRegistry } from '../../src/registries/FabricEnvironmentRegistry';
 
 import * as chai from 'chai';
-import { FabricEnvironmentRegistryEntry } from '../../extension/registries/FabricEnvironmentRegistryEntry';
-import { TestUtil } from '../TestUtil';
-import { FabricRuntimeManager } from '../../extension/fabric/FabricRuntimeManager';
-import { FabricRuntimeUtil } from 'ibm-blockchain-platform-common';
+import * as chaiAsPromised from 'chai-as-promised';
+import * as path from 'path';
+import { FabricEnvironmentRegistryEntry } from '../../src/registries/FabricEnvironmentRegistryEntry';
+import { FabricRuntimeUtil } from '../../src/util/FabricRuntimeUtil';
 
 chai.should();
+chai.use(chaiAsPromised);
 
 describe('FabricEnvironmentRegistry', () => {
 
     const registry: FabricEnvironmentRegistry = FabricEnvironmentRegistry.instance();
 
     before(async () => {
-        await TestUtil.setupTests();
+        const registryPath: string = path.join(__dirname, './tmp/registries');
```

Comment:
`path.join(__dirname, './tmp/registries')` should be `path.join(__dirname, 'tmp', 'registries')`